### DISCUSSION TITLE
how to load or import an nginx configuration already created by hand?

### DISCUSSION THREAD

**Discussion by tradexsrl** [0xJacky/nginx-ui#510] [Q&A]
as per title , i need to know how to import or how to map the already made nginx configuration to the UI
let me know , thank's

**Comment by 0xJacky** [comment]
Check this for more detail: https://nginxui.com/guide/getting-started.html#before-use

**Reply by tradexsrl** [reply]
i'm using rocky linux so i add to the nginx.conf these lines:
include /etc/nginx/conf.d/*.conf;
include /etc/nginx/sites-enabled/*;

first one was already present , restarted the vm , tried again your UI , log are readed as before , config file are listed under manage config but under manage sites - site list , i get Configuration file is test successful but there is nothing 
i can add that i have file called virtual_server.conf , which hold all subdomain proxied and a file called ssl_parameter.conf under /etc/nginx/conf.d/
in /sites-enabled/ i have nothing

**Reply by tradexsrl** [reply]
Any news on this problem please?

**Reply by 0xJacky** [reply]
Also, you need to create a folder `/etc/nginx/sites-available` and move all the servers.conf into it, then you can see the sites in Site List page.

**Reply by tradexsrl** [reply]
Ok solved thanks
